### DISCUSSION TITLE
Need help with LFG(

### DISCUSSION THREAD

**Discussion by pureamf** [TrinityCore/TrinityCore#30353] [Q&A]
Hi, everybody. Please help me!!! I don't understand what I need to do or change to enter the dungeon alone .What command should be executed? .Debug lfg doesn't work( it basically doesn't exist) could you please elaborate. I will be very grateful to you(
(war within)

**Comment by Aokromes** [comment]
you cannot without custom patches.

**Reply by pureamf** [reply]
What can I do? I just need to be able to go into dungeons and go through them alone. It's just getting boring gameplay without it. Maybe you have some edits to make?

**Reply by pureamf** [reply]
I'm just new to this and don't understand what I can do I've been sitting for 4 days looking for information(

**Reply by Aokromes** [reply]
> What can I do? I just need to be able to go into dungeons and go through them alone. It's just getting boring gameplay without it. Maybe you have some edits to make?

then walk alone, not using lfg.

**Reply by pureamf** [reply]
you wrote about custom patches, where can i get them?
